app/routes/planet_routes.py

```diff
@@ -0,0 +1,75 @@
+from flask import Blueprint, abort, make_response, request, Response
+from ..models.planet import Planet
+from .route_utilities import validate_model
+from ..db import db
+
+bp = Blueprint("planet_bp", __name__, url_prefix="/planets")
+
+@bp.post("")
+def create_planet():
+    request_body = request.get_json()
+
+    try:
+        new_planet = Planet.from_dict(request_body)
+
+    except KeyError as e:
+        response = {"message": f"Invalid request: missing {e.args[0]}"}
+        abort(make_response(response), 400)
+
+    db.session.add(new_planet)
+    db.session.commit()
+
+    response = new_planet.to_dict()
+    return response, 201
+
+@bp.get("")
+def get_all_planets():
+    query = db.select(Planet)
+
+    name_param = request.args.get("name")
+    if name_param:
+        query = query.where(Planet.name.ilike(f"%{name_param}%"))
+
+    description_param = request.args.get("description")
+    if description_param:
+        query = query.where(Planet.description.ilike(f"%{description_param}%"))
+
+    num_of_moons_param = request.args.get("num_of_moons")
+    if num_of_moons_param:
+        query = query.where(Planet.num_of_moons >= num_of_moons_param)
+
+    query = query.order_by(Planet.num_of_moons)
```

Comment:
This isn't something you need to refactor, but I could definitely see the name_param and the description_param being similar across models. I wonder if there is a way these two queries could be extracted to a helper function in your route_utilities.py!